tools/bin/hawqregister

```diff
@@ -469,8 +481,8 @@ class HawqRegister(object):
             self.filepath = self.files[0][:self.files[0].rfind('/')] if self.files else ''
             check_file_not_folder()
             check_database_encoding()
-            if self.mode != 'force' and self.mode != 'repair':
-                if not create_table():
+            if self.mode != 'repair':
+                if not create_table() and self.mode != 'force':
```

Comment:
I just want to create table if not exist in --force mode 
And if mode is --force we donot change the it
if not create_table() and self.mode != 'force':
        self.mode = 'usage2_table_exist'